AppBidding-MoPub/AppBidding-MoPub.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We do not need `AppBidding-MoPub.xcworkspace`.

Reply:
Done